### ISSUE TITLE
[Feature Request] Ability to search manga from recommendations tab through pinned sources at the same time

### ISSUE THREAD

**Issue by DottyPottyLotty** [jobobby04/TachiyomiSY#210]
## Why/User Benefit/User Problem
I am sorry if the title is confusing but here please bear with me. Currently for each manga page there is a button call 'Recommendation' which recommends similar manga.
![Screenshot_2021-02-13-17-09-05-63](https://user-images.githubusercontent.com/78997291/107843393-c0580400-6e1e-11eb-8663-b62aeca96864.jpg)
![Screenshot_2021-02-13-17-08-53-45](https://user-images.githubusercontent.com/78997291/107843406-ccdc5c80-6e1e-11eb-87bf-bab9d51b5680.jpg)
For this example I will pick the first manga Arifureta which normally will lead to this:
![Screenshot_2021-02-13-17-08-56-05](https://user-images.githubusercontent.com/78997291/107843420-f1d0cf80-6e1e-11eb-9998-46228696c60c.jpg)
I believe that rather than having pick a source to search it from, it will be great for it to act like the global search that automatically search from the pinned sources which will be faster than having to manually pick each different sources as not every source will have the manga.

## What/Requirements
Essentially the way this feature will behave the same way as how a global search would with the condition that it will only search within the pinned sources. Something like this:
![Screenshot_2021-02-13-17-09-44-66](https://user-images.githubusercontent.com/78997291/107843453-45431d80-6e1f-11eb-9eba-e537f9e84602.jpg)

**Comment by molitar** [comment]
I agree because the changes in recommendations method is crap from the old method!  It use to show recommended manga ON the source you were searching for.. now it recommends stuff that you can't even find!  No way in hell is ANYONE going to be manually searching 20+ extensions to find the damn recommendation!

I hope that commit does not take months as right now the most used feature to find new stuff to read is totally and utterly broken and useless.

**Comment by jobobby04** [comment]
It literally hasn't changed at all
